mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreen.tsx

```diff
@@ -63,6 +63,10 @@ import {
 import { useToast } from "react-native-toast-notifications";
 import { useDisplayUtxoWarning } from "@hooks/wallet/DisplayUtxoWarning";
 import { DEXIcon as SwapTokenButton } from "@screens/WalletNavigator/assets/BottomNavIcon";
+import {
```

Comment:
any reason for this change?

Reply:
Yes, I moved the `TokenDropdownButton` component into the common components folder, so need to replace the import